### ISSUE TITLE
Add independent connection points

### ISSUE THREAD

**Issue by comonoidial** [wandernauta/viskell#197]
Now connection points/anchor are always part of the in/outputs of a block.
Being able to place extra connection points elsewhere is useful for routing wires.
And would it allow for localized duplicating of values/wires, instead of all tracing back to the source block.

**Comment by wandernauta** [comment]
An `(id)` block would do this, though, right?

**Comment by comonoidial** [comment]
Yes an (id) block can be used as a workaround, but that is not pretty nor subtle.
So consider this as a lower priority nice to have feature request.
